python/lib/communication/dmod/communication/maas_request/ngen/partial_realization_config.py

```diff
@@ -51,7 +51,7 @@ def default_is_env_workaround(cls, v, values):
             return v
 
         def has_indicator(field_name: str):
-            return values.get(field_name) and values[field_name].split(cls._FROM_ENV_DELIMIT)[0] == cls._FROM_ENV_PREFIX
+            return values.get(field_name, ) and values[field_name].split(cls._FROM_ENV_DELIMIT)[0] == cls._FROM_ENV_PREFIX
```

Comment:
is this intentional?

Reply:
No - I don't know why that file was ever accessed in the first place. Good cache. Just undid the change.

Reply:
Looks like there might have been an odd find and replace. I've already found another instance of it, so I'm going to crawl through and fix any others that I find.